pkg/registry/apis/provisioning/repository/utils.go

```diff
@@ -0,0 +1,40 @@
+package repository
```

Comment:
Using "I put everything I don't know where to put in utils" is not a great practice to follow because it's not very concrete and it's hard later to find what things are about. This one is about workflows. So let's call this file `workflows.go` and give it a second thought next time we don't know where to put things. It's normally an small mental exercise :)

Reply:
that is exactly why i named it utils! 😛